static/css/login.css

```diff
@@ -3,82 +3,289 @@
     --light-green: #86A873;
     --blue-gray: #70798c;
 }
-body{
-    background-color: #f9f9f9;
-}
 
-.container {
-    display: flex;
-    justify-content: space-between;
+*, *:before, *:after{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Nunito', sans-serif;
```

Comment:
the icon next to profile in the header (the one that looks like a person) is not available in either of these fonts, so it won't look right (it won't be there at all until the thing i mentioned in the last comment is addressed though)

to get it to show up just add 'Font Awesome 6 Free' to the end of this list of fonts